### PR TITLE
Updated example plugins to not clobber Deploy() variables

### PR DESCRIPTION
**Pull request by meganokeefe**
This was my fault -- when I wrote the new example plugins (and goodbye, which I wrote weeks beforehand), I was having `Status()` and `Update()` calls return blank variables to sigma, which overwrites the result of deploy.

### CODE REVIEW

File: examples/coffee-run/coffee-orders/main.go [hunk 1]
```diff
@@ -87,15 +87,21 @@ func (c *CoffeeOrders) destroy(op pc.PluginArg) error {
...
+
 	if err := c.writeOutputs(true); err != nil {
 		return err
 	}
 	return nil
 }
 
 func (c *CoffeeOrders) update(op pc.PluginArg) error {
-	fmt.Println("CoffeeOrders Update")
+	fmt.Println("CoffeeOrders Deploy")
```
Comment (mikeynap) [inline examples/coffee-run/coffee-orders/main.go]: don't you mean update

Comment (adamschaub) [reply examples/coffee-run/coffee-orders/main.go]: It's deploying the update

File: examples/coffee-run/coffee-prices/main.go [hunk 2]
```diff
@@ -114,23 +114,27 @@ func (c *CoffeePrices) destroy(op pc.PluginArg) error {
 }
 
 func (c *CoffeePrices) status(op pc.PluginArg) error {
-	fmt.Println("CoffeePrices Status")
+	fmt.Println("CoffeePrices Deploy")
```
Comment (adamschaub) [inline examples/coffee-run/coffee-prices/main.go]: Status

File: examples/coffee-run/coffee-prices/main.go [hunk 3]
```diff
@@ -114,23 +114,27 @@ func (c *CoffeePrices) destroy(op pc.PluginArg) error {
...
+	fmt.Printf("Generated coffee prices: %v, TOTAL: %v", c.Vars.CoffeePrices, c.Vars.Total)
 	if err := c.writeOutputs(true); err != nil {
 		return err
 	}
 	return nil
 }
 
 func (c *CoffeePrices) update(op pc.PluginArg) error {
-	fmt.Println("CoffeePrices Update")
+	fmt.Println("CoffeePrices Deploy")
```
Comment (adamschaub) [inline examples/coffee-run/coffee-prices/main.go]: Update

**Pull request merged**